JSON/Example-Type-sourceOrDestination,measurement,bizTransaction.jsonld

```diff
@@ -0,0 +1,147 @@
+{
+  "@context": {
```

Comment:
TODO: has the EPCIS context embedded inline (because of context deployment delays), must be changed to a link